data/snippets.ts

```diff
@@ -553,6 +553,13 @@ const coding = [
     keyword: "css-ac",
     type: "template",
   },
+  {
+    name: "DOM Query Selector",
+    id: nanoid(),
+    text: `document.querySelector({cursor})`,
+    keyword: "ds",
```

Comment:
what does `ds` stand for?

Reply:
"Dom Selector" but it's personal i guess, might not make sense to everyone 😅

maybe i can change it to `qs` for "Query Selector"?

Reply:
haha ah gotcha! yeah people can change it to what they prefer. but I agree `qs` is slightly easier to infer!

Reply:
done, thanks for reviewing and feedback!